Library/Homebrew/cask/lib/hbc/cli/internal_stanza.rb

```diff
@@ -68,10 +65,17 @@ def run
             next
           end
 
-          if artifact_name && !value.key?(artifact_name)
-            opoo "no such stanza '#{artifact_name}' on Cask '#{cask}'" unless quiet?
-            puts ""
-            next
+          if value.nil? || (value.respond_to?(:to_a) && value.to_a.empty?) ||
+             (value.respond_to?(:to_s) && value.to_s == "{}") ||
+             (artifact_name && !value.include?(artifact_name))
+
+            if artifact_name
+              thing = artifact_name
```

Comment:
Can you find a better name than `thing`? Also, use a ternary for this.

Reply:
Ok!